### PR TITLE
Clean up bebugging code

### LINKED ISSUE #7841: Pub/Sub: System test failures on Python 2.7 (invalid syntax)
https://source.cloud.google.com/results/invocations/e8706e96-9a37-40d5-8411-7cd62206b4aa/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fpubsub/log

```
tests/unit/pubsub_v1/subscriber/test_subscriber_client.py:18: in <module>
    from google.cloud.pubsub_v1 import subscriber
google/cloud/pubsub_v1/__init__.py:19: in <module>
    from google.cloud.pubsub_v1 import subscriber
google/cloud/pubsub_v1/subscriber/__init__.py:17: in <module>
    from google.cloud.pubsub_v1.subscriber.client import Client
google/cloud/pubsub_v1/subscriber/client.py:30: in <module>
    from google.cloud.pubsub_v1.subscriber._protocol import streaming_pull_manager
E     File "/tmpfs/src/github/google-cloud-python/pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py", line 198
E       print(f"{messages_percent}, {bytes_percent}")
E                                                  ^
E   SyntaxError: invalid syntax
```

Looks like `streaming_pull_manager.py` is using a format string, which isn't valid Python 2.7 syntax. Needs to use `"{}, {}".format(...)`, instead.

### PR DESCRIPTION
**Pull request by anguillanneuf**
Cleanup bebugging code from #7758.  

This closes #7841.

### CODE REVIEW

File: pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def maybe_pause_consumer(self):
             if self._consumer is not None and not self._consumer.is_paused:
                 _LOGGER.debug("Message backlog over load at %.2f, pausing.", self.load)
                 self._consumer.pause()
-                print('paused')
+
```
Comment (plamut) [inline pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py]: Double line spacing between class methods, I think this is what caused the lint check to fail - just run `black` on it and should be fixed.

Comment (anguillanneuf) [reply pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py]: I tried `nox -s lint` with the proposed change. This file still failed, without any useful description.

**Comment by anguillanneuf** [comment]
I don't have permission to merge. Could someone do it for me? And could I be given the right permission?

**Comment by tswast** [comment]
@anguillanneuf , merged. You should have permission, though, according to https://github.com/googleapis/sloth/pull/210

@busunkim96 @JustinBeckwith can you help determine what's going wrong with Tianzi's permissions?

**Comment by busunkim96** [comment]
@JustinBeckwith Could you run sloth please?

**Pull request merged**